src/Microsoft.CodeAnalysis.Analyzers/Core/MetaAnalyzers/PreferIsKindAnalyzer.cs

```diff
@@ -75,6 +77,12 @@ private static void HandleBinaryOperation(OperationAnalysisContext context, Dict
             var possibleInvocation = operation.LeftOperand.WalkDownConversion();
             if (possibleInvocation is IConditionalAccessOperation conditionalAccess)
             {
+                // We don't currently report on Nullable<SyntaxToken>. If we'll report that in the future, the codefix must behave correctly.
+                if (SymbolEqualityComparer.Default.Equals(conditionalAccess.Operation.Type.OriginalDefinition, nullableOfT))
```

Comment:
IOperation APIs are not fully nullable annotated. I'm not sure if `Operation` or `Type` can ever be null or not.